scripts/docker-integration-test.sh

```
@@ -64,21 +69,27 @@ api_get_body() {
+    -X POST -H "Content-Type: application/json" ${csrf:+-H "X-CSRF-Token: $csrf"} \
+    -d "$2" "$API_URL$1" 2>/dev/null || echo "000"
 }
 
 api_put() {
+  local csrf; csrf=$(get_csrf_token)
   curl -s -o /dev/null -w "%{http_code}" --connect-timeout 5 --max-time 10 \
     -b "$COOKIE_JAR" -c "$COOKIE_JAR" \
-    -X PUT -H "Content-Type: application/json" -d "$2" "$API_URL$1" 2>/dev/null || echo "000"
+    -X PUT -H "Content-Type: application/json" ${csrf:+-H "X-CSRF-Token: $csrf"} \
+    -d "$2" "$API_URL$1" 2>/dev/null || echo "000"
 }
```

Shellcheck (0.11.0)

[info] This function is never invoked. Check usage (or ignored if invoked indirectly).

(SC2329)